### ISSUE TITLE
0.43.5 - NPE When Resolving Scenario

### ISSUE THREAD

**Issue by Flibjib** [MegaMek/mekhq#559]
When attempting to finish resolving a scenario, whether through the post-scenario prompts or the "Resolve Scenario" button, it fails to produce the popup.

Attaching logs, campaign and .mul from scenario
[Wolven Winter30370803.cpnx.zip](https://github.com/MegaMek/mekhq/files/1568235/Wolven.Winter30370803.cpnx.zip)
[Wolven Winter.mul.zip](https://github.com/MegaMek/mekhq/files/1568236/Wolven.Winter.mul.zip)
[mekhqlog.txt](https://github.com/MegaMek/mekhq/files/1568237/mekhqlog.txt)

**Comment by HammerGS** [comment]
Error Code from log.
```
Exception in thread "AWT-EventQueue-0" java.lang.IllegalArgumentException: invalid range properties
	at javax.swing.DefaultBoundedRangeModel.<init>(DefaultBoundedRangeModel.java:102)
	at javax.swing.JSlider.<init>(JSlider.java:274)
	at mekhq.gui.dialog.ResolveScenarioWizardDialog.initComponents(ResolveScenarioWizardDialog.java:485)
	at mekhq.gui.dialog.ResolveScenarioWizardDialog.<init>(ResolveScenarioWizardDialog.java:218)
	at mekhq.gui.BriefingTab.resolveScenario(BriefingTab.java:520)
	at mekhq.gui.BriefingTab.lambda$initTab$13(BriefingTab.java:298)
```

**Comment by Qwertronix** [comment]
Should be fixed by https://github.com/Qwertronix/mekhq/commit/fe3c3394d4e94b2a8d38db1cd5f16119fa2050ed. I'll send up a PR once I've also figured out what I broke in #560

**Comment by Kendricke** [comment]
Can't resolve two scenarios in a row.  One that existed under 43.4 (which I GM removed) and another which is under a brand new contract in 43.5.  Let me know if you want files to confirm this fix and whether or not we're getting a fast fix pushed on this since I apparently can't play t all under 43.5.

**Comment by Qwertronix** [comment]
I don't know whether a hotfix will be sent out (not up to me) but I reproduced the bug on my end in test files, and it's fully fixed with #561 as far as I can tell.

**Comment by Flibjib** [comment]
Would be so great to get a hot fix out, was looking forward to seeing all the improvements in my campaign
